llama_index/vector_stores/elasticsearch.py

```diff
@@ -379,6 +430,38 @@ def query(
         Raises:
             Exception: If Elasticsearch query fails.
 
+        """
+        return asyncio.get_event_loop().run_until_complete(
+            self.aquery(query, custom_query, es_filter, **kwargs)
+        )
+
+    async def aquery(
+        self,
+        query: VectorStoreQuery,
+        custom_query: Optional[
+            Callable[[Dict, Union[VectorStoreQuery, None]], Dict]
+        ] = None,
+        es_filter: Optional[List[Dict]] = None,
+        **kwargs: Any,
+    ) -> VectorStoreQueryResult:
+        """Asynchronous query index for top k most similar nodes.
+
+        Args:
+            query_embedding (List[float]): query embedding
```

Comment:
nit - needs to be updated to `query (VectorStoreQuery) ...`

Reply:
nice catch!